ci/linkcheck.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -ev
+set -o pipefail
+
+# https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then # running in cron job
+  FLAGS=""
+
+  echo "Doing full link check."
+elif [ "$CI" = "true" ] ; then # running in PR CI build
+  if [ -n "$TRAVIS_COMMIT_RANGE" ]; then
```

Comment:
I'm pretty sure this is backwards:

```suggestion
  if [ -z "$TRAVIS_COMMIT_RANGE" ]; then
```